autoscheduler/scraper/models/instructor.py

```diff
@@ -1,4 +1,10 @@
 from django.db import models
 
 class Instructor(models.Model):
-    """ Describe model here """
+    """ Model that represents a professor, who may teach many courses """
+    id = models.IntegerField(primary_key=True)
+    email_address = models.CharField(max_length=320, null=True)
+    name = models.CharField(max_length=64)

```

Comment:
Fun thing I found out when searching for instructors longest names: the instructor with the longest names' name is: `Kalu Arachchillage Senarath Dhananjaya Bandara Dharmasena`

Reply:
57 < 64

Reply:
Oh no yeah this line is fine, I just thought it was funny lmao

Reply:
Also, I think pablo Picasso's full name would also throw an error, but he's not a prof here, so I think 64 is a reasonable limit